Docs/source/running_cpp/parameters.rst

```diff
@@ -1042,6 +1042,12 @@ Numerics and algorithms
 
     Note that in the current implementation in WarpX these 3 numbers must be equal.
 
+* ``interpolation.galerkin_scheme`` (`0` or `1`)
+    Whether to use a Galerkin scheme when gathering fields to particles.
+    When set to `1`, the interpolation order for each field component is reduced by one along the direction of that component.
+    For example, `E_z` is gathered using ``interpolation.nox``, ``interpolation.noy``, and ``interpolation.noz - 1``.
+    Defaults to `1` unless ``warpx.do_nodal = 1`` and/or ``algo.field_gathering = momentum-conserving``.
```

Comment:
In https://github.com/ECP-WarpX/WarpX/pull/1171#discussion_r457665194 (PR that was replaced by this one) @RemiLehe suggested:
> Could you also add a reference to the papers that are cited https://www.sciencedirect.com/science/article/pii/S0021999113002556 (reference 18 to 20)?

I agree that it would be good to cite those references in the docs. You can add linked references in reStructuredText (rst) format with the following syntax:
```rst
(`Godfrey and Vay, 2013 <https://doi.org/10.1016/j.jcp.2013.04.006>`_)
```
This will display, for example, the text "(Godfrey and Vay, 2013)", with the content inside brackets linked to the external URL. If you want to link more references I would suggest this style (chronological order, use "et al" for more than two authors):
```rst
(`Author1, 1980 <https://doi.org/url1>`_; `Author2 and Author3, 2000 <https://doi.org/url2>`_; `Author4 et al, 2020 <https://doi.org/url3>`_)
```

I believe that this is better than what is mostly done in the current WarpX docs, where you will find mainly plain text references in the text (no links, neither internal nor external) plus a list of the same references at the bottom of each page (with external links).

Reply:
Reference added. It looks like full details are actually in reference 17 of [Godfrey and Vay, 2013], but I couldn't find a URL for it. Is linking one and mentioning the other good enough?

Reply:
Yes, thanks for adding this.